tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
Can you please explain the intent/purpose/need of this change.

Reply:
I was pretty sure I removed this. Someone from the packages made this change to tsconfig. I haven't found the culprit and haven't even been able to reproduce the problem on my main machine, but at the moment this change is definitely not necessary. I'll remove this.

Reply:
Ha, no worries, makes sense.